src/main/features/response/views/check-and-send.njk

```diff
@@ -283,7 +285,7 @@
       {{ tableEnd() }}
     {% endif %}
 
-    {% if statementOfMeansIsApplicable and draft.statementOfMeans %}
+    {% if statementOfMeansIsApplicable and draft.statementOfMeans and not draft.defendantDetails.partyDetails.isBusiness() %}
```

Comment:
I think the additional checks needs to be in `StatementOfMeansFeature.isApplicableFor()` and we should only rely on `statementOfMeansIsApplicable` flag here.

Reply:
`StatementOfMeansFeature.isApplicableFor()` was updated for the sake of the task list, but murtuza explicitly wanted nothing changed on the check-and-send page - hence the exclusion. We either have the exclusion here or an inclusion on the task list; I deemed this simpler...

Confirm?